packages/block-library/src/cover/controls.native.js

```diff
@@ -68,14 +68,9 @@ function Controls( {
 		[ minHeight ]
 	);
 
+	const [ availableUnits ] = useSettings( 'spacing.units' );
 	const units = useCustomUnits( {
```

Comment:
Not from this PR, but this appears to be so repetitive! I wonder if we should abstract it to a separate hook with these default args.